internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -150,9 +146,9 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 			},
 			"cluster_id": schema.StringAttribute{ // TODO: was generated as id
 				Computed: true,
-				PlanModifiers: []planmodifier.String{
-					stringplanmodifier.UseStateForUnknown(),
-				},
+				// PlanModifiers: []planmodifier.String{
+				// 	stringplanmodifier.UseStateForUnknown(),
```

Comment:
in fact I would reevaluate where we use `UseStateForUnknown` and try to limit them

Reply:
Added to https://jira.mongodb.org/browse/CLOUDP-286481